src/DynamoCore/Models/NodeModel.cs

```diff
@@ -1671,21 +1638,6 @@ private void OnNodeValueQueried(AsyncTask asyncTask)
         /// tessellation divisions to use for regenerating render packages.</param>
         public void RequestVisualUpdateAsync(IScheduler scheduler, EngineController engine, IRenderPackageFactory factory)
         {
-            //if (Workspace.DynamoModel == null)
-            //    return;
-
-            // Imagine a scenario where "NodeModel.RequestVisualUpdateAsync" is being 
-            // called in quick succession from the UI thread -- the first task may 
-            // be updating '_renderPackages' when the second call gets here. In 
-            // this case '_renderPackages' should be protected against concurrent 
-            // accesses.
-            // 
-            lock (RenderPackagesMutex)
-            {
-                renderPackages.Clear();
-                HasRenderPackages = false;
-            }
-
             RequestVisualUpdateAsyncCore(scheduler, engine, factory);
```

Comment:
It's not possible to be internal, because it's called from the view model, which is in a different assembly.